grep/README.md

```diff
@@ -0,0 +1,28 @@
+# Grep
+
+The `grep` command line tool is used for searching and matching text files contained in the regular expressions.
+
+
+## Running the grep Command Using [`run-app-elfloader`](https://github.com/unikraft/run-app-elfloader)
+
+Change directory into `run-app-elfloader` and run a similar command to the following:
+
+```console
+./run.sh -r path/to/dynamic-apps/grep /bin/grep [OPTION] [PATTERN] [FILENAME]
+```
```

Comment:
Give a snippet of what the output of the `run.sh` will be, both here and in the linux run section.